schemas/2014-04-01-preview/Microsoft.Sql.json

```diff
@@ -169,6 +170,54 @@
         "properties"
       ],
       "description": "Microsoft.Sql/firewallrules"
+    },
+    "administratorsChild": {
```

Comment:
>administratorsChild [](start = 5, length = 19)

The definitions for nested resource types should be put in "definitions" section. It would be nice if you can also fix for "firewallrulesChild" and "databasesChild".